src/rmq/CMakeLists.txt

```diff
@@ -38,6 +42,17 @@ install(
         COMPONENT librmq-dev
 )
 
+install(EXPORT rmqcppTargets 
+        FILE rmqcppTargets.cmake 
+        DESTINATION ${CMAKE_INSTALL_PREFIX}/share/rmqcpp 
+        NAMESPACE rmqcpp::)
```

Comment:
We decided to go with `rmqcpp::rmq` be how users link to rmqcpp, with `rmqcpp::rmqtestmocks` a possibility in future.